ISISComputingGroup/ibex_gui
# Connecting to p vs

###Description of work

Refactors several classes necessary for the OPI view to appear in the device screens to be factored out of the synoptic class into a separate location to facilitate reuse.

Sets up a token OPI view which is currently locked on to Linkam 95 in the device screens perspective. This is a proof of concept to show that an OPI can now be shown in the device screens view entirely decoupled from the synoptic view

###To test

1. Open IBEX
1. Navigate to the device screens view
1. Verify that an OPI is displayed

###Acceptance criteria

1. OPI displayed in device screens view
1.Current synoptic OPI functionality is unaffected
1. Device screens plugin still independent of synoptic plugins

As the changes to the view are entirely token to prove the refactor was successful, I haven't added any system tests. I expect these will be added once the GUI is properly implemented.
